Gifski/ConversionViewController.swift

```diff
@@ -116,19 +118,15 @@ final class ConversionViewController: NSViewController {
 			progress?.cancel()
 		}
 
-		stopConversion { [weak self] in
-			// It's safe to force-unwrap as there's no scenario where it will be nil.
-			let viewController = AppDelegate.shared.previousEditViewController!
-			viewController.isConverting = false
-
-			self?.push(viewController: viewController)
+		stopConversion { [self] in
+			self.presentingViewController?.dismiss(self)
 		}
 	}
 
 	private func didComplete(conversion: Gifski.Conversion, gifUrl: URL) {
 		let conversionCompleted = ConversionCompletedViewController(conversion: conversion, gifUrl: gifUrl)
-		stopConversion { [weak self] in
```

Comment:
Why are you removing `weak`?

Reply:
This was an incorrect fix for a bug where the delayed tooltip on ConversionCompletedViewController could throw an error under unlikely user behaviour. Steps to reproduce:

1. Start a conversion
2. Hit escape exactly during the transition from ConversionViewController to ConversionCompletedViewController (spamming escape as soon as transition begins) 
3. Click back button

I have found a correct fix for this and will update shortly